### ISSUE TITLE
ACT w/discrete action space

### ISSUE THREAD

**Issue by dirkmcpherson** [tonyzhaozh/act#32]
Hi, I'm trying to use ACT as a baseline for an imitation learning / MBRL paper. The simplest environment I have uses a discrete action space. Is there any reason to believe that if I make the standard continuous -> discrete changes (listed below) ACT wont perform? 

Discrete -> Continuous Changes
1. Action head outputs logits and actions are sampled from a categorical distribution
2. Cross entropy loss between logits and demonstrated action indexes

thanks for this great work!
